packages/checkout-com/src/useCko.ts

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable camelcase, @typescript-eslint/camelcase */
 
 import { createContext } from './payment';
-import { Configuration } from './configuration';
+import { Configuration, getSaveInstrumentKey } from './configuration';
 import { ref } from '@vue/composition-api';
+import { CKO_PAYMENT_TYPE } from './helpers';
 import useCkoCard from './useCkoCard';
 
 const error = ref(null);
 const availableMethods = ref([]);
+const contextId = ref(null);
```

Comment:
Type here would be helpful

Reply:
It is string. However, you do not have access to it.